### PR TITLE
update cdh and elk with dockerfile locations and docker pull command

### PR DESCRIPTION
**Pull request by omhpe**
Updated KD CDH and ELK with Dockerfile location and docker pull command in respective docs

### CODE REVIEW

File: deploy/example_catalog/docs/cdh632-readme.md [hunk 1]
```diff
@@ -117,4 +117,14 @@ After changing the member count of  the workers and reapplying the cluster,check
...
+Adding Kerberos support
+
+
+#### Docker Image location
+
+* docker.io/bluedata/cdh632multi
+
+#### Docker Pull command
+
+* docker pull docker.io/bluedata/cdh632multi:1.4
```
Comment (kmathur2) [inline deploy/example_catalog/docs/cdh632-readme.md:129]: did we fix something in 1.4 tag?

File: deploy/example_catalog/docs/elk771-readme.md [hunk 2]
```diff
@@ -92,4 +92,12 @@ More about the ELK cluster can be read [here.](https://www.elastic.co/what-is/el
...
\ No newline at end of file
+   Similarly we can perform update, delete on the product indice by sending rest request from the console tool in Kibana.
+
+#### Docker Image location
+
+* docker.io/bluedata/elk771
+
+#### Docker Pull command
+
+* docker pull docker.io/bluedata/elk771:1.1
```
Comment (kmathur2) [inline deploy/example_catalog/docs/elk771-readme.md:103]: same question here for 1.1 tag

File: deploy/example_catalog/docs/cdh632-readme.md (hunk 1 above)
Comment (kmathur2) [reply deploy/example_catalog/docs/cdh632-readme.md:129]: or is it just documentation

File: deploy/example_catalog/docs/elk771-readme.md (hunk 2 above)
Comment (omhpe) [reply deploy/example_catalog/docs/elk771-readme.md:103]: @kmathur2 The change is only in the documentation with the latest tag of the elk image which is 1.1.

File: deploy/example_catalog/docs/cdh632-readme.md (hunk 1 above)
Comment (omhpe) [reply deploy/example_catalog/docs/cdh632-readme.md:129]: @kmathur2 The change is only in the documentation with the latest tag of the cdh image which is 1.4.

**Review by kmathur2 — APPROVED** [review]

**Pull request merged**